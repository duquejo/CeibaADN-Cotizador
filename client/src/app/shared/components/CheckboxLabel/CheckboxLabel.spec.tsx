import * as React from 'react';
import { render, mount, ShallowWrapper, shallow } from 'enzyme';
import { CheckBoxLabel } from './index';


describe('Pruebas sobre componente <CheckBoxLabel />', () => {

    let componentWrapper: ShallowWrapper;

    beforeEach( () => {
        componentWrapper = shallow(<CheckBoxLabel/>);
    });

    it('DeberÃ­a renderizar el elemento', () => {
        expect( componentWrapper ).toMatchSnapshot();
    });
});