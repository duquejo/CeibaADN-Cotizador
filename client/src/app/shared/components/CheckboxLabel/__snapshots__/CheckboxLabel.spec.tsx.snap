// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas sobre componente <CheckBoxLabel /> Deber√≠a renderizar el elemento 1`] = `
<label
  className="sc-bdvvtL hKVzWT"
/>
`;
